recipes-kernel/linux/files/bcm2712-raspberrypi5-xen.dtso

```diff
@@ -33,24 +33,10 @@
  * HACK uart regs to be xen page aligned to avoid XEN fail in dom0less mode
  * when vuart "vpl011" is used for DomU
  */
-&_uart0 {
+&uart10 {
 	reg = <0x7d001000 0x1000>;
 };
 
-/*
```

Comment:
remove hack should be done in a separate commit as well. IMO. But it's up to you